install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1941,6 +1941,14 @@ envoy:
   # -- DNS policy for Cilium envoy pods.
   # Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
   dnsPolicy: ~
+  debug:
+    admin:
+      # -- Enable admin interface for cilium-envoy.
+      # This is useful for debugging and should not be enabled in production.ss
```

Comment:
some trailing `s` here as well. also mentioned by the docu check. 

> Documentation/helm-values.rst:1088: (ss)  Enable admin interface for cilium-envoy. This is useful for debugging and should not be enabled in production.ss

Reply:
oops :(